README.md

```diff
@@ -72,6 +72,14 @@ Autotools is a common standard on Linux, and can be used for MacOS.
 Instructions are
 [provided here](https://github.com/libtcod/libtcod/tree/master/buildsys/autotools).
 
+Conan, the first that you should be is install conan: `sudo pip3 install conan`,
+the second step is added a remote used for libtcod: 
+`conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan`,
+the third step is use conan in the project: 
+`conan install . -s build_type=Debug --install-folder=cmake-build-debug`,
+the fourth step is: `cd cmake-build-debug` and then: `cmake ..`, finally: `make`
```

Comment:
Ultimately the main problem with these instructions are that they're for a non-standard use of Conan.  The readme in the samples directory already shows how to use Conan in an external project. You build libtcod through Conan by including it externally, not by building it internally.

Reply:
I don't have the vaguest idea of the correct use of Conan, but due to the current structure of libtcod, I think it's much more natural to include the library internally rather than externally, because in any case, the examples are part of the project, otherwise we would have a separate repository to hold the examples.